File: glucare_java/glucare/src/main/java/com/uade/glucare/dto/UserDTO.java
package com.uade.glucare.dto;

public class UserDTO {
        private final String nombre;
        private final String correoElectronico;
        private final int edad;
    
        public UserDTO(String nombre, String correoElectronico, int edad) {
            this.nombre = nombre;
            this.correoElectronico = correoElectronico;
            this.edad = edad;
        }
    
        public String getNombre() {
            return nombre;
        }
    
        public String getCorreoElectronico() {
            return correoElectronico;
        }
    
        public int getEdad() {
            return edad;
        }
}

File: glucare_flutter/lib/model/UserDTO.dart
class UserDTO {
  final String nombre;
  final String correoElectronico;
  final int edad;

  UserDTO({
    required this.nombre,
    required this.correoElectronico,
    required this.edad,
  });

  factory UserDTO.fromJson(Map<String, dynamic> json) {
    return UserDTO(
      nombre: json['nombre'],
      correoElectronico: json['correoElectronico'],
      edad: json['edad'],
    );
  }

  Map<String, dynamic> toJson() {
    return {
      'nombre': nombre,
      'correoElectronico': correoElectronico,
      'edad': edad,
    };
  }
}
